### PR TITLE
[msys2] - ofxGui avoid fs::path / string implicit conversion. fix for __builtin_va_arg_pack issue in swprintf

### PR DESCRIPTION
**Pull request by dimitre**
it can close 
- https://github.com/openframeworks/openFrameworks/issues/8220

### CODE REVIEW

File: libs/openFrameworks/communication/ofSerial.cpp [hunk 1]
```diff
@@ -446,7 +446,7 @@ bool ofSerial::setup(string portName, int baud){
 		cfgSize = sizeof(cfg);
 		GetCommConfig(hComm, &cfg, &cfgSize);
 		int bps = baud;
-		swprintf(buf, L"baud=%d parity=N data=8 stop=1", bps);
+		swprintf(buf, 80, L"baud=%d parity=N data=8 stop=1", bps);
```
Comment (ofTheo) [inline libs/openFrameworks/communication/ofSerial.cpp:450]: this is the only thing needed for the fix right?

**Comment by ofTheo** [comment]
fine to merge with me - as long as the failing checks were failing before :)

File: libs/openFrameworks/communication/ofSerial.cpp (hunk 1 above)
Comment (dimitre) [reply libs/openFrameworks/communication/ofSerial.cpp:450]: For the specific fix yes, but the system complained about string - fs::path conversion here, so I had to change other things to be able to find this error. if you prefer I can split this PR in two and merge this specific one

Comment (ofTheo) [reply libs/openFrameworks/communication/ofSerial.cpp:450]: ah cool! - no that's all good - thanks for the fix @dimitre !

**Pull request merged**